agents-and-function-calling/bedrock-agents/features-examples/13-create-agent-using-CDK/lambda/kb_sync/.dockerignore

```diff
@@ -0,0 +1,3 @@
+.dockerignore
```

Comment:
Docker dependency needs to be clear in the readme as per-requesite

Reply:
Docker dependency is already removed.